play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/FormGroup.scala

```diff
@@ -34,14 +36,30 @@ object FormGroup {
     (
       (__ \ "classes").readNullable[String] and
         (__ \ "attributes").readWithDefault[Map[String, String]](Map.empty) and
-        (__ \ "afterInput").readNullable[Content]
+        (__ \ "afterInput").readNullable[Content] and
+        (__ \ "beforeInput").readNullable[Content]
     )(FormGroup.apply _)
 
   implicit def jsonWrites: OWrites[FormGroup] =
     (
       (__ \ "classes").writeNullable[String] and
         (__ \ "attributes").write[Map[String, String]] and
-        (__ \ "afterInput").writeNullable[Content]
+        (__ \ "afterInput").writeNullable[Content] and
+        (__ \ "beforeInput").writeNullable[Content]
     )(unlift(FormGroup.unapply))
 
+  def jsonReadsCheckboxes: Reads[FormGroup] = (
+    (__ \ "classes").readNullable[String] and
+      (__ \ "attributes").readWithDefault[Map[String, String]](Map.empty) and
+      (__ \ "afterInputs").readNullable[Content] and
+      (__ \ "beforeInputs").readNullable[Content]
+  )(FormGroup.apply _)
+
+  def jsonWritesCheckboxes: OWrites[FormGroup] =
+    (
+      (__ \ "classes").writeNullable[String] and
+        (__ \ "attributes").write[Map[String, String]] and
+        (__ \ "afterInputs").writeNullable[Content] and
+        (__ \ "beforeInputs").writeNullable[Content]
+    )(unlift(FormGroup.unapply))
```

Comment:
I reckon we should make this something like jsonReadsForMultipleInputs / jsonWritesForMultipleInputs

because (although they have no examples at the moment) we should also use it for govukdate and govukradios formgroups - in case they add examples in the future

https://github.com/search?q=repo%3Aalphagov%2Fgovuk-frontend%20beforeInputs&type=code